src/app/portlets/content-types/fields/content-type-fields-variables/content-type-fields-variables.component.html

```diff
@@ -0,0 +1,32 @@
+<form [formGroup]="form" (ngSubmit)="addVariableForm($event)" #formEl="ngForm" (keyup.enter)="formEl.ngSubmit" novalidate>
```

Comment:
`addVariableForm($event)` you are sending a param that the method doesn't expect.